distributed/protocol/tests/test_serialize.py

```diff
@@ -88,8 +88,8 @@ def test_serialize_bytestrings():
         assert bb == b
 
 
-def test_serialize_empty_array():
-    a = array("I")
+@pytest.mark.parametrize("a", [array("I"), memoryview(bytearray())])
+def test_serialize_empty_objs(a):
```

Comment:
This test was handling something fairly specific. Think it would better for us to break this out into a separate test. Going to propose some suggestions to handle this and the other issue